.github/workflows/CI-pipeline.yml

```diff
@@ -97,24 +97,23 @@ jobs:
         with:
           endpoint: claudie
 
-      # Login to Docker Hub
-      - name: Login to DockerHub
-        if: (needs.check-changes.outputs.ARRAY_OF_CHANGES != '' && github.event.pull_request.draft == false)
+      - name: Set up docker
         uses: docker/login-action@v2
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN}}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
       # Build the new images that were changed by a recent commit - tag image latest as well for testing
       - name: Build and push new images
         if: (needs.check-changes.outputs.ARRAY_OF_CHANGES != '' && github.event.pull_request.draft == false)
         run: |
-          arr=(${{ needs.check-changes.outputs.ARRAY_OF_CHANGES }})
-          for path in "${arr[@]}"
+          ARR=(${{ needs.check-changes.outputs.ARRAY_OF_CHANGES }})
+          for SERVICE in "${ARR[@]}"
           do
-            echo "-----Building $path-----"
-            IMGTAG="${{secrets.DOCKERHUB_USERNAME}}/$path:${SHORT_GITHUB_SHA}-${GITHUB_RUN_NUMBER}" 
-            DOCKER_BUILDKIT=1 docker buildx build --push --platform linux/amd64,linux/arm64 --tag $IMGTAG -f ./services/$path/Dockerfile . 
+            echo "-----Building $SERVICE-----"
+            IMGTAG="ghcr.io/berops/claudie/$SERVICE:${SHORT_GITHUB_SHA}-${GITHUB_RUN_NUMBER}" 
```

Comment:
Just wondering are also going to include the release version in the image tag ?

Reply:
The release version is handled in [release.yml](https://github.com/Berops/claudie/blob/master/.github/workflows/release.yml), more specifically [here](https://github.com/Berops/claudie/blob/feat/migrate-dh-to-ghcr/.github/workflows/release.yml#L65).